cluster/log-dump/log-dump.sh

```diff
@@ -106,7 +106,7 @@ function log-dump-ssh() {
 function copy-logs-from-node() {
     local -r node="${1}"
     local -r dir="${2}"
-    local files=( ${3} )
+    read -r -a files <<< "${3}"
```

Comment:
Wouldn't you want something like

`while IFS= read -r line; do files+=("$line"); done <<< "$3"`

here to populate the `files` array?